cmd/maintenance.go

```diff
@@ -18,10 +18,10 @@ func (d DeisCmd) MaintenanceInfo(appID string) error {
 		return err
 	}
 
-	if *appSettings.Maintenance {
-		d.Println("Maintenance mode is on.")
+	if appSettings.Maintenance == nil || !*appSettings.Maintenance {
+		d.Println("Maintenance mode is disabled.")
```

Comment:
On and off is fine with me, I was just trying to standardize between `deis routing` and `deis maintenance`. Changing back...